### ISSUE TITLE
For frame-ancestor allow both the named IP and also numeric IP

### ISSUE THREAD

**Issue by Joe-Winchester** [zowe/zowe-install-packaging#532]
**Problem**   
Customer who was unable to resolve to a named host but could use numeric IP was unable to launch the explorers because of a frame-ancestor error

**Proposed solution**   
In the frame-ancestors add a numeric IP host as well as an alphabetic host name

**Comment by yann-59** [comment]
We had a similar problem ... we had 3 DNS depending the site we connected. If we used the DNS's name provided at the install, we had no error. If we used alternate DNS, the explorer wont' launch and we can see an error message :  
Refused to display 'https://xxxx.yyyy.toto:7554/ui/v1/explorer-mvs/' in a frame because an ancestor violates the following Content Security Policy directive: "frame-ancestors xxxx.yyyy.toto:*".
In previous version of Zowe , we modified the index.html to add the good name :
 è,                                          
 "csp": é                                    
   "frame-ancestors": Ý"https://xxxx.yyyyy:*"   
Now, is it a way to add multiple DSN or IP address for using explorer with ZLUX?
